src/git.js

```diff
@@ -50,8 +50,12 @@ module.exports = function gitFactory(repository = '', origin = 'origin') {
             return git(repository)
                 .remote(['get-url', origin])
                 .then( url => {
-                    const parsed = gitUrlParse(url.trim());
-                    return parsed.protocol === 'file' ? parsed.pathname : parsed.full_name;
+                    try {
+                        const parsed = gitUrlParse(url.trim());
+                        return parsed.protocol === 'file' ? parsed.pathname : parsed.full_name;
+                    } catch(e) {
+                        return url;
+                    }
```

Comment:
I guess there was some parsing error - can we please log it, or it will be impossible to debug issues.

Reply:
it fails in unit tests for local path

Reply:
Suggestion: add exactly that comment, so the next developer understands